app/components/identity-steps/step-four.hbs

```diff
@@ -0,0 +1,65 @@
+<div data-test='chaincode' class='chaincode-page'>
+  <h3 data-test='heading' class='chaincode-page__heading heading__h3'>
+    Chaincode Generation
+  </h3>
+  <p data-test='description' class='chaincode-page__description'>
+    A private key that you need to use in your profile service URL and deploy
+    for source that you're the source of the URL
+  </p>
+  {{#if this.isChaincodeClicked}}
+    <div data-test='chaincode-container' class='chaincode-container'>
+      <div
+        data-test='chaincode-container__value'
+        class='chaincode-container__value'
+      >
+        {{#if this.hideChaincode}}
+          **************

```

Comment:
this should not be hardcoded Shubham, `*` should get generated as per the number of characters inside the input field if you are trying to simulate a similar thing as happens with input:type=password field

Reply:
Yes because not integrated with backend

Reply:
it is not input field...we know how many digit of chaicode is created